.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
That's probably a documentation error... https://github.com/travis-ci/travis-ci/issues/977#issuecomment-14873652